src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStreamAsyncMemoryOverloads.cs

```diff
@@ -139,21 +140,7 @@ private void AnalyzeCompilationStart(CompilationStartAnalysisContext context)
                 ParameterInfo.GetParameterInfo(cancellationTokenType)
             };
 
-            // Create the arrays with the exact parameter order of the desired methods
```

Comment:
I noticed these were unused in the analyzer code. The preferred method extraction did not require these arrays.